src/repos/FlyerRepo.ts

```diff
@@ -70,6 +70,18 @@ const getFlyersByCategorySlug = async (
     });
 };
 
+/**
+ * Looks for unique category slugs for all the Flyers
+ * @returns a list of Strings with all the category names. 
+ */
+const getAllFlyerCategories = async (
+): Promise<String[]> => {
+  return FlyerModel.collection.distinct("categorySlug")
+    .then((categories) => {
+      return categories;
+    });
```

Comment:
The `.then` method is not needed here. When the `FlyerModel.collection.distinct` call finishes, the `.then` returns a new promise, whose behavior is defined based on the function you put in. But your function just returns the resulting value, so it is redundant. See [here](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Promise/then#return_value) for more info.